### DISCUSSION TITLE
MetaMask Deeplink

### DISCUSSION THREAD

**Discussion by wpcorbett79** [WalletConnect/walletconnect-monorepo#4304] [Developer Support] [answered]
Hi, I can't seem to get MetaMask to open through deeplink on mobile. Do you know the correct link format? I'm using below but it received a 404 error on Android and redirects to app store on iOS.
https://metamask.app.link/dapp/https://easnft.com

**Comment by glitch-txs** [comment, marked as answer]
For in-app browser deeplink you should contact MetaMask dev support. For WalletConnect one you can use [Web3Modal](https://docs.walletconnect.com/web3modal/about) or [EthereumProvider](https://docs.walletconnect.com/advanced/providers/ethereum) which will handle this for you internally.

If you'd like to go more low level you can use the [explorer API](https://docs.walletconnect.com/cloud/explorer)
